Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizationDemo.cs

```diff
@@ -44,7 +44,7 @@ private void Awake()
         {
             markerParent = new GameObject();
             markerParent.name = "Boundary Demo Markers";
-            markerParent.transform.parent = CameraCache.Main.transform.parent;
+            markerParent.transform.parent = MixedRealityManager.Instance.MixedRealityPlayspace.transform;
```

Comment:
Why not just make the `MixedRealityPlayspace` a transform?

Reply:
Agreed